11ty/definitions/blacklist-whitelist.md

```diff
@@ -0,0 +1,63 @@
+---
+title: blacklist whitelist
+slug: blacklist-whitelist
+flag:
+  level: avoid
+  text: racist term
+defined: true
+excerpt: 'umbrella term used to represent a list of entities/people who are to be allowed or denied access to a service, privilege, or recognition: to "blacklist" is "to exclude", to "whitelist" means "to include".'
+speech: noun; verb
+alt_words:
+  blacklist (noun):
+    - blocklist
+    - denylist
+    - excludelist
+  whitelist (noun):
+    - allowlist
+    - passlist
+    - trustlist
+    - safelist
+  to blacklist (verb):
+    - to block
+    - to deny
+    - to exclude
+  to whitelist (verb):
+    - to allow
+    - to trust
+    - to include
+reading:
+  - text: UK security agency bans blacklist and whitelist for being racist
+    href: https://www.techspot.com/news/85080-uk-cyber-security-agency-stop-using-racist-blacklist.html
+  - text: Google Chrome team moves away from black/whitelist to be more inclusing
+    href: https://9to5google.com/2020/06/12/google-android-chrome-blacklist-blocklist-more-inclusive/
+  - text: Google developers styleguide entry for blacklist
+    href: https://developers.google.com/style/word-list#blacklist
+  - text: '"Blacklists” and “whitelists": a salutary warning concerning the prevalence of racist language in discussions of predatory publishing'
+    href: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6148600/
+---
+
+To allow or deny access to a service, a privilege, or recognition.
+
+## Issues
+
+<!-- Black/white admittance and denial (Jim Crow) -->
+
+The first problem is the admittance and denial of people based on their race and skin color. The term are rooted in racism: "blacklist" started appearing when the enslavement and deportation of Africans by Europeans began.
+
+Using "black" and "white" to determine whether an entity is included or excluded from a group is correlated to the racist idea that Black and White people have been respectively denied and allowed privileges based on the color of their skin. Similarly to the concept of [master/slave](/definitions/master-slave), blacklist/whitelist is used casually in the realm of software development, which reinforces the exclusion of Black people from the industry.
+
+<!-- Black/white as a good/bad dichotomy -->
+
+The second issue with these terms is the implication that black is bad and white is good. A "blacklist" often refers to a list of undesirable things (spammy phone numbers, undesired guests, etc.) whereas a "whitelist" refers to a list of virtuous things (important guests, emails we want to receive, etc.) In 1610, the _Douglas Harper Etymology Dictionary_ defined the term "blacklist" as a "list of persons who have incurred suspicion".
+
```

Comment:
```suggestion
While the origins predate the transatlantic slave trade, the concept of a blacklist as eliciting "suspicion" reinforce the systemic racism that causes disproportionately [higher incarceration and incrimination of Black people](https://www.washingtonpost.com/graphics/2020/opinions/systemic-racism-police-evidence-criminal-justice-system/) for equivalent crimes to non-Black people. 
```
Adding a clause such as this will help to prevent the "gotchas" I shared above.